itest/rust/src/callable_test.rs

```diff
@@ -0,0 +1,91 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use godot::bind::{godot_api, GodotClass};
+use godot::builtin::{varray, Callable, ToVariant, Variant};
+use godot::engine::{Object, RefCounted};
+use godot::obj::{Base, Gd, Share};
+use godot::prelude::GodotString;
+use godot::test::itest;
+
+#[derive(GodotClass)]
+#[class(init, base=RefCounted)]
+struct CallableTestObj {
+    #[base]
+    base: Base<RefCounted>,
+
+    value: i32,
+}
+
+#[godot_api]
+impl CallableTestObj {
+    #[func]
+    fn foo(&mut self, a: i32) {
+        self.value = a;
+    }
+
+    #[func]
+    fn bar(&self, b: i32) -> GodotString {
+        b.to_variant().stringify()
+    }
+}
+
+#[itest]
+fn callable_validity() {
+    let obj = Gd::<CallableTestObj>::new_default();
+    assert!(!obj.callable("doesn't_exist").is_valid());
+    assert!(!obj.callable("doesn't_exist").is_null());
+    assert!(!obj.callable("doesn't_exist").is_custom());
+    assert!(obj.callable("foo").is_valid());
+    assert!(Callable::default().is_null());
+    assert!(!Callable::default().is_custom());
```

Comment:
Could you also check all 3 methods for these 2 cases?

I.e. we would have 3x3 assert statements in total.

Reply:
im also adding `.object().is_some()/is_none()`